src/odemis/gui/cont/tabs.py

```diff
@@ -508,20 +508,31 @@ def load_data(self, data):
         # Create streams from data
         streams = data_to_static_streams(data)
 
-        # TODO: Clear previous overview streams
-        self.clear_data()
-
         for s in streams:
+            self.clear_overview_data(s)
             scont = self._overview_stream_controller.addStream(s, add_to_view=True)
             scont.stream_panel.show_remove_btn(True)
 
-    def clear_data(self):
+    def clear_live_streams(self):
+        """
+        Clear the content of the live streams
+        """
+        live_streams = [stream for stream in self.tab_data_model.streams.value if isinstance(stream, LiveStream)]
+        for stream in live_streams:
+            stream.image.value = None
```

Comment:
That's not enough to reset the data. For instance, if you change the brightness/contrast, the stream will update the .image, and it'll be back from the dead!
See how to do it in gui.cont.microscope._reset_streams(). Essentially:
```python
            if s.raw:
                s.raw = []
                s.image.value = None
                s.histogram._value = numpy.empty(0)
                s.histogram.notify(s.histogram._value)
```